### ISSUE TITLE
Consider enrolling on proper translation infrastructure

### ISSUE THREAD

**Issue by bragefuglseth** [ADBeveridge/raider#149]
As a member of GNOME Circle, File Shredder is eligible for enrollment in GNOME's translation system (and also expected to be connected to some sort of translation infra, even if not GNOME's). The process is pretty straightforward, see [here](https://gitlab.gnome.org/Infrastructure/damned-lies/-/issues/543) for an example.

**Comment by ADBeveridge** [comment]
yea i don't have it on gitlab, which seems to be a requirement.

**Comment by bragefuglseth** [comment]
The example I linked to is hosted on GitHub. The GNOME sysadmin team has a bot user that handles it 🙂

**Comment by ADBeveridge** [comment]
Oh interesting! I'll look into it this weekend

**Comment by ADBeveridge** [comment]
I'm going to be quite busy till this summer, then i'll work on enrollment.

**Comment by ADBeveridge** [comment]
https://gitlab.gnome.org/Infrastructure/damned-lies/-/issues/671

**Comment by MatejKovacic** [comment]
Hej, just to let you kniw that there is some Anubis challenge on this site (to block AI companies from scrapping), which would block me, if I visit site with Vanadium browser on GraphneneOS. Firefox on GOS works.

**Comment by ADBeveridge** [comment]
Yeah, the Gnome team is using anubis for their gitlab (they had issues with too many requests). fortunately the actual translations won't be on gitlab (as far as I know). so it should be fine.

**Comment by ADBeveridge** [comment]
https://l10n.gnome.org/module/raider/
